mantidproject/mantid
# Vesuvio - update test values to peak height and bin index

The unit and system tests for `VesuvioPreFit`, `VesuvioTOFFit` and `VesuvioCorrections` have all been updated to check the value for bin index as well as peak height as this data is more meaningful than checking the first and last bin. These tests have been run on Windows 7 and Ubuntu 14.04 and build servers should check RHEL7 (system tests) and RHEL6 unit tests

**To test:**
- Ensure the unit and system tests pass on the build servers.
- Code review to ensure the changes seem like a sensible change for the tests

Fixes #16065

_Concerns testing therefore does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
